### PR TITLE
Create onelife.thedev.me.json

### PR DESCRIPTION
**Pull request by GifPpro**

### CODE REVIEW

**Comment by YazeedT** [comment]
It will not, and your records are totally incorrect and invalid. Please read the README carefully before submitting a pull request.

**Comment by YazeedT** [comment]
I have added support for NS records. Feel free to go through the README and update your changes to become valid.

**Comment by 1efan** [comment]
Ok, I’m new to this stuff.

**Comment by YazeedT** [comment]
Feel free to ask questions here or send an email to contact@thedev.me and I can walk you through it, and we'll see if we can make it work. 

The records you added to AAAA are NS records, which I believe is common to be provided for a game server.

You can start by editing the JSON to only include the NS records your host gave you, and we'll go from there.

**Comment by 1efan** [comment]
Okay, I added the NS records only.

File: domains/onelife.thedev.me.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      "192.0.2.1",
+      "192.0.2.2"
+    ],
+    "AAAA": [
+      "2001:db8::1",
+      "2001:db8::2"
+    ],
+    "CNAME": [
+      "example.com"
+    ],
```
Comment (YazeedT) [inline domains/onelife.thedev.me.json:19]: Delete.

**Comment by YazeedT** [comment]
Submit a new pull request after following the instructions in the README.

**Comment by GifPpro** [comment]
I updated the json file, is that what you meant by delete the other records if not needed?

**Pull request closed without merging**